crates/wash-lib/Cargo.toml

```diff
@@ -29,7 +29,7 @@ cli = [
     "path-absolutize",
 ]
 nats = ["async-nats", "wadm"]
-build-adapters = [ "wasmcloud-component-adapters/build-adapters" ]
+build-adapters = ["wasmcloud-component-adapters/build-adapters"]
```

Comment:
I'd try to avoid changing this line since https://github.com/wasmCloud/wasmCloud/pull/1146 touches it too

Reply:
Gah, I don't even have a formatter installed anymore, how does this happen